### PR TITLE
Options to filter manifests or reports modes

### PR DESCRIPTION
**Pull request by gildub**
Closes https://github.com/fusor/cpma/issues/310

### CODE REVIEW

File: pkg/env/env.go [hunk 1]
```diff
@@ -93,7 +93,17 @@ func setConfigLocation() (err error) {
 }
 
 func surveyMissingValues() error {
-	err := surveyConfigSource()
+	err := surveyManifests()
+	if err != nil {
+		return err
+	}
+
```
Comment (arapov) [inline pkg/env/env.go]: Hey, we agreed to write such statement as:
``
if err := surveyManifests(); err != nil {
    return err
}
``
a while back. Could you change these before merge please.
See https://github.com/fusor/cpma/blob/d28c4c1fd45f70af783a20488ed8865c1169b27a/pkg/env/env.go#L46 for example.

**Comment by arapov** [comment]
Also, we want Travis to succeed :)

File: pkg/transform/api_transform_test.go [hunk 2]
```diff
@@ -24,6 +24,7 @@ var loadAPIExtraction = func() transform.APIExtraction {
 }()
 
 func TestAPIExtractionTransform(t *testing.T) {
+	t.Skip()
```
Comment (alexander-demichev) [inline pkg/transform/api_transform_test.go]: Is this needed?

Comment (gildub) [reply pkg/transform/api_transform_test.go]: Nope that's a leftover. Thanks!

File: pkg/env/env.go (hunk 1 above)
Comment (gildub) [reply pkg/env/env.go]: Yes you're right, this is actually legacy code we need to change. Thanks

File: pkg/env/env.go [hunk 3]
```diff
@@ -159,6 +165,44 @@ func surveyConfigSource() error {
 	return nil
 }
 
+func surveyManifests() error {
+	manifests := viperConfig.GetString("Manifests")
+	if !viperConfig.InConfig("manifests") {
```
Comment (alexander-demichev) [inline pkg/env/env.go]:
```suggestion
	if !viperConfig.InConfig("manifests") && manifests == "" {
```

File: pkg/env/env.go [hunk 4]
```diff
@@ -159,6 +165,44 @@ func surveyConfigSource() error {
...
+		} else {
+			viperConfig.Set("Manifests", false)
+		}
+	}
+	return nil
+}
+
+func surveyReports() error {
+	reports := viperConfig.GetString("Reports")
+	if !viperConfig.InConfig("reports") {
```
Comment (alexander-demichev) [inline pkg/env/env.go]:
```suggestion
	if !viperConfig.InConfig("reports") && reports == "" {
```

**Pull request merged**